src/instructlab/model/evaluate.py

```diff
@@ -1,23 +1,28 @@
 # Standard
+import logging
 import os
-import subprocess
-import time
 
 # Third Party
 from instructlab.eval.evaluator import Evaluator
 from instructlab.eval.mmlu import MMLUBranchEvaluator, MMLUEvaluator
 from instructlab.eval.mt_bench import MTBenchBranchEvaluator, MTBenchEvaluator
 import click
 
+# Local
+from ..utils import http_client
+
+logger = logging.getLogger(__name__)
+
 BENCHMARK_TO_CLASS_MAP = {
     "mmlu": MMLUEvaluator,
     "mmlu_branch": MMLUBranchEvaluator,
     "mt_bench": MTBenchEvaluator,
     "mt_bench_branch": MTBenchBranchEvaluator,
 }
 
-# TODO: Remove after proper vLLM integration
-# pylint: disable=consider-using-with
+JUDGE_MODEL_NAME = "judge_model"
+TEST_MODEL_NAME = "test_model"
+BASE_TEST_MODEL_NAME = "base_test_model"
```

Comment:
I think we should be passing these down via the config - @cdoern could you chime in here?

Reply:
Noting that these are not configurable values by the user.

Reply:
Dan and I talked about these values, and realized it's not necessary for the user to need to set this. The values are just needed to make the test run. The user shouldn't care what the name assigned to the model is (not the path or type of model) during the a run of MT-Bench.. 

Instead of introducing unneeded config options and flags we decided to just hard code them.

Reply:
Okay, fine by me!